_posts/2018-05-08-CocoaPods-1.6.0-beta.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "CocoaPods 1.6.0 Beta Begins!"
+author: dnkoutso
+categories: cocoapods releases
+---
+
+_CocoaPods 1.6.0_ next release will focus on stability, performance and scalability for large projects through a longer beta cycle.
+
+<!-- more -->
+
+We are happy to announce that the first beta of 1.6.0 is out! For this release, we are revamping the build settings generation to be faster and easier to maintain. We've also improved integration of test specs for pods that have multple ones.
+
+### Revamped Build Settings
+
+The generation of build settings has been completely re-written. The primary goals for this re-write was to clean up the code but also allow this process to scale better on large projects. Having to apply a fix would often cause a cascading amount of performance regressions making it very hard to test but also apply the fix for everyone.
```

Comment:
> Having to apply a fix would often...

Probably worth stating who would be the applying the fixes

> When contributors to CocoaPods make build setting changes it would often cause a cascading amount of performance regressions. This makes it very hard to test but also to be sure the fix applies for everyone.